lib/components/SSpinner.vue

```diff
@@ -2,7 +2,7 @@
   <svg
     class="SSpinner"
     xmlns="http://www.w3.org/2000/svg"
-    viewBox="0 0 100 100"
+    viewBox="22 22 55 55"
```

Comment:
This might break some stuff. Without cropping the svg and increasing its width/height using css, it is pushing the input box down (layout shift).

Reply:
Good catch. I'll add special treatment for `SSpinner`. Indeed this is going to have effect in  many places so 😅  We can bypass by doing something like below in `<SInputBase>`.

```css
.check-icon.SSpinner {
  margin: -4px;
  width: 24px;
  height: 24px;
}
```